scripts/README.md

```diff
@@ -0,0 +1,217 @@
+
+Converts GitHub flavored markdown changelogs to release notes.
+
+### [pod_lib_lint.rb](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/pod_lib_lint.rb)
+
+Wrapper script for running `pod lib lint` tests to include dependencies from the monorepo.
+
+### [release_testing_setup.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/release_testing_setup.sh)
+
+Utility script for the release workflow.
+
+### [remove_data.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/remove_data.sh)
+
+Cleanup script for CI workflows.
+
+### [run_database_emulator.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/run_database_emulator.sh)
+
+Run the RTDB emulator.
+
+### [run_firestore_emulator.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/run_firestore_emulator.sh)
+
+Run the Firestore emulator.
+
+### [setup_bundler.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_bundler.sh)
+
+Set up the Ruby bundler.
+
+### [setup_check.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_check.sh)
+
+Install tooling for the check workflow.
+
+### [setup_quickstart.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_quickstart.sh)
+
+Set up a QuickStart for integration testing.
+
+### [setup_quickstart_framework.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_quickstart_framework.sh)
+
+Set up a QuickStart for zip distribution testing.
+
+### [setup_spm_tests.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_spm_tests.sh)
+
+Configuration for SPM testing.
+
+### [spm_test_schemes](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/spm_test_schemes)
+
+Schemes used by above script to enable test target schemes.
+
+### [test_archiving.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/test_archiving.sh)
+
+Test Xcode Archive build.
+
+### [test_catalyst.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/test_catalyst.sh)
+
+Test catalyst build.
+
+### [test_quickstart.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/test_quickstart.sh)
+
+Test QuickStart.
+
+### [test_quickstart_framework.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/test_quickstart_framework.sh)
+
+Test QuickStart with the zip distribution.
+
+### [update_xcode_target.rb](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/update_xcode_target.rb)
+
+Script to add a file to an Xcode target.
+
+### [xcresult_logs.py](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/xcresult_logs.py)
+
+Tooling used by `build.sh` to get the log output for an `xcodebuild` invocation.
+
+### [zip_quickstart_test.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/zip_quickstart_test.sh)
+
+Run the tests associated with a QuickStart.
```

Comment:
Are these specific to the zip build of a quickstart, I assume?

Reply:
yes. fixed